### ISSUE TITLE
Never draggable elements

### ISSUE THREAD

**Issue by backbone87** [clauderic/dnd-kit#1993]
I have a table with draggable rows. Some rows might have their dragging disabled and setting `disabled` on `useDraggable` works well. It also has good a11y because the drag handle can still receive focus so the user can be informed why the row is currently not draggable. However the table also has some rows that are never draggable and in this case I do not want to have a drag handle visually and neither the main ref nor the drag handle should be able to receive focus.

Consider the following code taken from my table (reduced for brevity):

```tsx
interface RowProps {
  canDrag?: 'never' | 'enabled' | 'disabled';
  // ...
}

function Row({ id, canDrag, canDrop, data, children }: RowProps): JSX.Element {
  const ref: Ref<HTMLTableRowElement> = useRef(null);
  const { handleRef } = useDraggable({
    // have to remove the element ref because when `canDrag === 'never'` the handleRef
    // is not attached and then the row would be made focusable
    element: canDrag === 'never' ? undefined : ref,
    id,
    data,
    disabled: canDrag !== 'enabled',
  });
  useDroppable({ element: ref, id, data, disabled: !canDrop, collisionDetector });

  return useMemo(
    () => (
      <StyledRow ref={ref} id={id}>
        <StyledDragCell 
          // have to remove the handleRef other wise the drag handle is made focusable even when `useDraggable({ disabled: true })`
          elementRef={canDrag === 'never' ? undefined : handleRef}
          data-can-drag={canDrag}
        >
          <ParallelDots />
        </StyledDragCell>
        {children}
      </StyledRow>
    ),
    [id, handleRef, canDrag, children],
  );
}
```

it would be very convenient to have something like an `inert` boolean on draggable to say: yeah while i set this draggable up, it can actually never be dragged and neither its element ref nor its handle ref should ever be set up with tab index or other attrs

example usage:
```tsx
interface RowProps {
  canDrag?: 'never' | 'enabled' | 'disabled';
  // ...
}

function Row({ id, canDrag, canDrop, data, children }: RowProps): JSX.Element {
  const ref: Ref<HTMLTableRowElement> = useRef(null);
  const { handleRef } = useDraggable({ 
    element: ref,
    id,
    data,
    disabled: canDrag !== 'enabled',
    inert: canDrag === 'never', // or alternatively pass something like `always` to `disabled`
  });
  useDroppable({ element: ref, id, data, disabled: !canDrop, collisionDetector });

  return useMemo(
    () => (
      <StyledRow ref={ref} id={id}>
        <StyledDragCell elementRef={handleRef} data-can-drag={canDrag}>
          <ParallelDots />
        </StyledDragCell>
        {children}
      </StyledRow>
    ),
    [id, handleRef, canDrag, children],
  );
}
```
